py/src/braintrust/logger.py

```diff
@@ -1752,7 +1784,7 @@ class Dataset(ObjectFetcher):
     You should not create `Dataset` objects directly. Instead, use the `braintrust.init_dataset()` method.
     """
 
-    def __init__(self, lazy_metadata: LazyValue[ProjectDatasetMetadata], version: Union[None, int, str] = None):
+    def __init__(self, lazy_metadata: LazyValue[ProjectDatasetMetadata], version: Union[None, int, str] = None, include_output_alias=True):
```

Comment:
if this is true, should we swap `expected` -> `output`? Otherwise if they `json.dumps` it'll have both fields.